Dockerfiles/midstream/entrypoint.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+main() {
+    local username="${SSH_AUTH_USER:-default}"
+    local home
+    home=$(mk_home "$username")
+    setup_passwd "$username" "$home"
+    exec sleep inf
+}
+mk_home() {
+    local username="${1:?}"
+    mkdir -p "/tmp/home/$username" && \
+    echo "/tmp/home/$username" && \
+    return 0
+    mktemp --tmpdir -d "${username}.XXXXXX"
+}
+setup_passwd() {
+    local username="${1:?}"
+    local home="${2:?}"
+    local uid="$(id -u)"
+    local gid="${4:-0}"
```

Comment:
I think it makes sense to change the `gid` to be the 3rd parameter as `uid` isn't tied to a parameter anymore.